coalib/parsing/CliParsing.py

```diff
@@ -116,9 +116,11 @@ def check_conflicts(sections):
         if (
                 section.get('no_config', False) and
                 (section.get('save', False) or
-                 section.get('find_config', False))):
+                 section.get('find_config', False) or
+                 (str(section.get('config', 'no_input')) != 'no_input'))):
```

Comment:
`no_input`? Isn't just `True`/`False` returned?

Reply:
@Makman2 I think I can not use `True/False` here becasue `section.get('config')` gives `''` instead of False when no config is present or passed through cli.

Reply:
What do you say?

Reply:
You think or you know? :D

Reply:
I know ;)